packages/pluggableWidgets/toggle-buttons-native/webpack.config.prod.js

```diff
@@ -1 +1,8 @@
-module.exports = require("../../../configs/webpack.prod.native.js");
+const merge = require("webpack-merge");
+const baseConfig = require("@mendix/pluggable-widgets-tools/configs/webpack.native.config.js");
+
+baseConfig[0].module.rules[2].use.options.plugins.push("@babel/plugin-transform-flow-strip-types");
+baseConfig[0].mode = "production";
```

Comment:
`const [baseConfig] = require("@mendix/pluggable-widgets-tools/configs/webpack.native.config.js");` then you dont need to use `baseConfig[0]` in 3 different places

Reply:
True, will change it

Reply:
👍

Reply:
Changed it.